src/NServiceBus.Gateway.RavenDB/NServiceBus.Gateway.RavenDB.csproj

```
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>$(SolutionDir)NServiceBus.snk</AssemblyOriginatorKeyFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="NServiceBus" Version="[7.2.1, 8.0.0)" />
+    <PackageReference Include="NServiceBus.Gateway" Version="[3.1.0, 4.0.0)" />
+    <PackageReference Include="Particular.Packaging" Version="0.5.0" PrivateAssets="All" />
```

I forgot ConfigureAwaits until last minute.
```suggestion
    <PackageReference Include="Particular.Packaging" Version="0.5.0" PrivateAssets="All" />
    <PackageReference Include="Particular.CodeRules" Version="0.2.1" PrivateAssets="All" />
```

---

Addressed